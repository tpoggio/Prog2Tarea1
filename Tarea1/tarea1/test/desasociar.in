esClave 15
desasociar 15
esClave 15
asociar 15 4.0
esClave 15
desasociar 15
esClave 15
asociar 15 4.0
esClave 15
valor 15
asociar 95 24.0
asociar 85 2.0
asociar 85 32.0
esClave 85
valor 85
asociar 65 5.0
asociar 25 67.0
asociar 35 -12.0
asociar 75 2.5
asociar 45 2.1
asociar 55 -24.0
desasociar 85
esClave 85
asociar 85 32.0
esClave 85
valor 85
asociar 5 32.0
esClave 5
valor 5
asociar 10 10.0
esClave 10
desasociar 35
esClave 35
asociar 10 10.0
esClave 10
valor 10
Fin
