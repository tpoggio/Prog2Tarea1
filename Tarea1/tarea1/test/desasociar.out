1>15 no es clave. 
2>Desasociar. 
3>15 no es clave. 
4>Asociar.
5>15 es clave. 
6>Desasociar. 
7>15 no es clave. 
8>Asociar.
9>15 es clave. 
10>15 está asociado a 4.00. 
11>Asociar.
12>Asociar.
13>Asociar.
14>85 es clave. 
15>85 está asociado a 2.00. 
16>Asociar.
17>Asociar.
18>Asociar.
19>Asociar.
20>Asociar.
21>Asociar.
22>Desasociar. 
23>85 no es clave. 
24>Asociar.
25>85 es clave. 
26>85 está asociado a 32.00. 
27>Asociar.
28>5 es clave. 
29>5 está asociado a 32.00. 
30>Asociar.
31>10 no es clave. 
32>Desasociar. 
33>35 no es clave. 
34>Asociar.
35>10 es clave. 
36>10 está asociado a 10.00. 
37>Fin.
